pkg/api-server/resource_endpoints.go

```diff
@@ -288,7 +291,15 @@ func (r *resourceEndpoints) validateResourceRequest(request *restful.Request, re
 	if r.descriptor.Scope == model.ScopeMesh && meshName != resourceMeta.Mesh {
 		err.AddViolation("mesh", "mesh from the URL has to be the same as in body")
 	}
-	err.AddError("", mesh.ValidateMeta(name, meshName, r.descriptor.Scope))
+	if create {
+		err.AddError("", mesh.ValidateMeta(resourceMeta, r.descriptor.Scope))
+	} else {
+		if verr, msg := mesh.ValidateMetaBackwardsCompatible(resourceMeta, r.descriptor.Scope); verr.HasViolations() {
+			err.AddError("", verr)
+		} else if msg != "" {
+			core.Log.Info(msg)
```

Comment:
should there be more context added to this log message?

Reply:
Now it looks like:
> [WARNING] Kuma 2.5.x introduces more strict validation rules for resource name, it must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character. You can't CREATE resources with invalid names anymore, but it's possible to UPDATE such resources until Kuma 2.7.x is released.

What would you like to add to this message?

Reply:
isn't it just alone? Should we not indicate which resource is triggering it?

Reply:
Right, let me add this